chatbot.py

```diff
@@ -162,4 +177,13 @@ def message_to_bot(H, clf, learn_response):
             if subj[0] == "geocoding" or subj[0] == location:
                 B = googleMapsApiModule.geocoding(location)
                 learn_response = LearnResponse.MESSAGE.name
+        if (len(subj) != 0 and subj[0] in ["elevation", "height", "depth"]) or (
+            len(adj) != 0 and adj[0] in ["high"]
+        ):
+            if compound_NNP:
+                location = " ".join(
+                    word for word in nltk.word_tokenize(H) if word in compound_NNP
+                )
```

Comment:
Proper Nouns with more than one word in them were seen having a dependency of '**compound**' from the results returned from `StanfordDependencyParser` from `parse_sentence()` in `utilities.py`.